### DISCUSSION TITLE
Publish the changlog of the new database on this Github DB page?

### DISCUSSION THREAD

**Discussion by LuigiP57** [PygmalionOfCyprus/cmo-db-requests#4861] [General]
Okay, I have to admit that the update on the latest database is a little too long, but I don't mean to rush it. I have noticed that when database updates are not available in a timely manner, we are likely to submit many duplicate tickets that we do not know have been completed. 

Therefore, I suggested that we might publish the changelog about the new database on the github page of this CMO DB instead of waiting for the official patch release.

**Comment by PygmalionOfCyprus** [comment]
I like it! It's also an easier place to go through and see all the DB changelogs in one place. I'll upload a bunch of them.

**Reply by LuigiP57** [reply]
Hope to see them tonight!
